src/main/java/org/opensearch/security/privileges/dlsfls/DocumentPrivileges.java

```diff
@@ -185,10 +185,10 @@ RenderedDlsQuery evaluate(PrivilegesEvaluationContext context) throws Privileges
 
     /**
      * This is a DLS query where any templates (like ${user.name}) have been interpolated and which has been
-     * succesfully parsed to a QueryBuilder instance.
+     * successfully parsed to a QueryBuilder instance.
      */
     public static class RenderedDlsQuery {
-        public static RenderedDlsQuery MATCH_NONE = new RenderedDlsQuery(new MatchNoneQueryBuilder(), "{\"match_none:\" {}}");
+        public static RenderedDlsQuery MATCH_NONE = new RenderedDlsQuery(new MatchNoneQueryBuilder(), "{\"match_none\": {}}");
```

Comment:
This was a typo in the existing code, but didn't fail any tests. Ideally, this typo should have been caught with tests in this repo.

Reply:
Very true. I will look into this.

Reply:
Ah, found it: The second param is only needed for the compat mode in mixed cluster scenarios:

https://github.com/opensearch-project/security/blob/00204dd49ae555856f13a319dd805e69a29c8ee2/src/main/java/org/opensearch/security/privileges/dlsfls/DlsFlsLegacyHeaders.java#L188

So, such issues would be only detected in BWC tests, which have only limited test cases at the moment. We can add such a test. Yet, we might want to wait for the results of the discussion at https://github.com/opensearch-project/security/issues/5181 because it might change the situation significantly.